docs/manual/common/concepts/ComponentTechnologies.md

```diff
@@ -0,0 +1,23 @@
+
+* Akka --- Lagom [[Persistence|PersistentEntity]], [[Publish-Subscribe|PubSub]], and [[Cluster|Cluster]] are implemented on top of [Akka](http://akka.io/), Lightbend's toolkit for building concurrent, distributed, and resilient message-driven applications. This is an implementation detail that does not directly concern you when developing simple microservices. However, advanced users can call some Akka APIs directly. Specifically:
+<ul> 
+   <li>If you want to scale your microservices out across multiple servers, Lagom provides clustering via [Akka Cluster](http://doc.akka.io/docs/akka/2.4/java/cluster-usage.html).</li>
+   <li>Customers of Lightbend's [Reactive Platform](https://www.lightbend.com/products/lightbend-reactive-platform) can additionally use [[Lightbend ConductR|ConductR]] and Akka's [Split Brain Resolver](http://doc.akka.io/docs/akka/akka-commercial-addons-1.0/java/split-brain-resolver.html) for deployment and cluster management and [Lightbend Monitoring](https://www.lightbend.com/products/monitoring) for monitoring production systems. See [[Cluster]] for more details.</li>
+    <li>As described in [[Implementing services|ServiceImplementation]], A Lagom service may be "simple" or "streamed".  Streaming, asynchronous Lagom services are built on top of [Akka Streams](http://doc.akka.io/docs/akka/2.4/java/stream/index.html).</li>
+    </ul>
+
+* Cassandra --- By default, Lagom microservices that need to persist data use the  [Cassandra](http://cassandra.apache.org) instance that runs as part of the development environment. You can also use an existing [[Cassandra Server|CassandraServer]] database or another type of database. See [[Managing data persistence|ES_CQRS]] for more information.   
+
+* Guice --- Like Play, Lagom uses [Guice](https://github.com/google/guice) for dependency injection.
+
+* Play Framework --- Lagom is implemented on top of [Play Framework](https://www.playframework.com), Lightbend's web framework. (Play is in turn built on the popular, standard [netty](http://netty.io) network transport library.) This is an implementation detail that does not directly concern you when developing simple microservices.  However, advanced users can call some Play APIs directly. If you have an existing Play Framework application to which you want to add microservices, Lagom provides support for that use case.
+
+* SLF4 & Logback --- Lagom uses [SLF4J](http://www.slf4j.org/) for logging, backed by [Logback](http://logback.qos.ch/) as its default logging engine. See [[Logging]] for more information.
+
+* Typesafe Config Library --- Lagom and many of its component technologies are configured using the [Typesafe Config](https://github.com/typesafehub/config) library.  The configuration file format is [HOCON](https://github.com/typesafehub/config/blob/master/HOCON.md), a powerful and expressive superset of JSON.
+
+* WebSockets --- Communication with browser-based clients is via [WebSockets](https://tools.ietf.org/html/rfc6455).
+
```

Comment:
I'm not sure this item applies here. Websockets is a standard each end implements. This is the only item on the list that is not a product _per se_.

Also, only _some_ browser-lagom communication uses websockets.